addons/website/models/website.py

```diff
@@ -113,6 +113,10 @@ def _default_logo(self):
 
     robots_txt = fields.Text('Robots.txt', translate=False, groups='website.group_website_designer')
 
+    recaptcha_public_key = fields.Char("Public Key")
```

Comment:
use same label than google to avoid customer doubts

site key & secret key

![image](https://user-images.githubusercontent.com/4848797/78567954-bab68880-7821-11ea-9d81-be5b90bdc783.png)